invenio_vocabularies/templates/semantic-ui/invenio_vocabularies/subjects.html

```diff
@@ -0,0 +1,25 @@
+{#
+  Copyright (C) 2020 CERN.
+  Copyright (C) 2020-2022 Northwestern University.
+  Copyright (C) 2021 New York University.
+
+  Invenio RDM Records is free software; you can redistribute it and/or modify
+  it under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+{%- set subjects = field_value -%}
+{% if subjects %}
+  <ul class="ui horizontal list no-bullets subjects">
+    {%- for subject in subjects %}
+    <li class="item">
+      {%- set q = 'metadata.subjects.subject:"' + subject.subject + '"' %}
```

Comment:
nit: you can use tilde (`~`) operator for string concatenation. See [docs](https://jinja.palletsprojects.com/en/latest/templates/#other-operators)